bump.js

```diff
@@ -62,18 +63,22 @@ module.exports = async function ({ context, inputs }) {
   )
 
   const prBody = getPRBody(_template(tpl), { newVersion, draftRelease, inputs })
-
-  await callApi(
-    {
-      method: 'POST',
-      endpoint: 'pr',
-      body: {
-        head: `refs/heads/${branchName}`,
-        base: context.payload.ref,
-        title: `${PR_TITLE_PREFIX} ${branchName}`,
-        body: prBody,
+  try {
+    await callApi(
+      {
+        method: 'POST',
+        endpoint: 'pr',
+        body: {
+          head: `refs/heads/${branchName}`,
+          base: context.payload.ref,
+          title: `${PR_TITLE_PREFIX} ${branchName}`,
+          body: prBody,
+        },
       },
-    },
-    inputs
-  )
+      inputs
+    )
+  } catch (err) {
+    await run('git', ['push', 'origin', '--delete', branchName])
```

Comment:
now this is more of a detail and surely we're not being so specific about error handling in other places, but for the sake of completeness it should be noted that this operation may also fail, in which case we wouldn't give any details to the user.